desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml

```diff
@@ -4,12 +4,23 @@
     <metadata_license>CC0-1.0</metadata_license>
     <project_license>MIT OR Apache-2.0</project_license>
     <name>Ruffle</name>
+    <name xml:lang="fr">Ruffle</name>
+    <name xml:lang="nl">Ruffle</name>
     <name xml:lang="pl">Ruffle</name>
+    <name xml:lang="zh-CN">Ruffle</name>
     <summary>Play Flash games &amp; movies</summary>
+    <summary xml:lang="fr">Jouez à des jeux et regardez des animations en Flash</summary>
+    <summary xml:lang="nl">Speel Flash spellen &amp; animaties</summary>
     <summary xml:lang="pl">Graj w gry i odtwarzaj filmy Flash</summary>
+    <summary xml:lang="ru">Проигрывайте Flash-игры &amp; ролики</summary>
+    <summary xml:lang="zh-CN">播放 Flash 游戏和动画</summary>
     <developer id="rs.ruffle">
         <name>The Ruffle Team</name>
+        <name xml:lang="fr">L'équipe de Ruffle</name>
+        <name xml:lang="nl">Het Ruffle Team</name>
         <name xml:lang="pl">Zespół Ruffle</name>
+        <name xml:lang="ru">Команда Ruffle</name>
+        <name xml:lang="zh-CN">The Ruffle Team</name>
```

Comment:
That seems like something that could be translated

Reply:
Sure, but I don't think this should be a blocker for merging this batch. It's a bit redundant to add it explicitly, as the fallback to the original English will be the same, but this isn't any less correct IMHO.